cli/cmd/plugin/unmanaged-cluster/config/config_test.go

```diff
@@ -203,8 +218,12 @@ func TestInitializeConfigurationFromConfigFile(t *testing.T) {
 		t.Errorf("expected ServiceCidr to be set to '9.9.9.0/24', was: %q", config.ServiceCidr)
 	}
 
-	if config.NumberOfNodes != "99" {
-		t.Errorf("expected NumberOfNodes to be set to 'bongos', was: %q", config.NumberOfNodes)
+	if config.ControlPlaneNodeCount != "99" {
+		t.Errorf("expected ControlPlaneNodeCount to be set to 'bongos', was: %q", config.ControlPlaneNodeCount)
```

Comment:
Error string on both of these looks like it was copied from the CNI assertion?

Reply:
Guilty as charged 😂